tests/src/rules/default.js

```diff
@@ -1,305 +1,229 @@
 import path from 'path';
-import { test, testVersion, SYNTAX_CASES, getTSParsers, parsers } from '../utils';
+import { test, testVersion, SYNTAX_CASES, parsers as utilsParsers, getTSParsers } from '../utils';
+import  parsers  from '../parsers';
 import { RuleTester } from 'eslint';
 import semver from 'semver';
 import { version as tsEslintVersion } from 'typescript-eslint-parser/package.json';
+import fromEntries from "object.fromentries"
+
+ 
 
 import { CASE_SENSITIVE_FS } from 'eslint-module-utils/resolve';
 
-const ruleTester = new RuleTester();
+
 const rule = require('rules/default');
 
+
+
+const parserOptions = {
+  ecmaVersion: 2018,
+  sourceType: 'module',
+  ecmaFeatures: {
+    jsx: true,
+  },
+
+  settings: {
+    "import/parsers": fromEntries(getTSParsers().map((parser)=>[parser, [".ts"]] ))
+     
```

Comment:
```suggestion
  settings: {
    'import/parsers': fromEntries(getTSParsers().map((parser) => [parser, ['.ts']]))
```
you may want to enable eslint autofixing in vscode :-)

Reply:
Thank you. I'll do that proptly